### PR TITLE
Make country codes consistent

### PR DESCRIPTION
**Pull request by fortuna**
How do I test this?
Is the CI test good enough?

### CODE REVIEW

**Comment by fortuna** [comment]
Ping

File: Android/app/src/main/java/app/intra/sys/firebase/AnalyticsWrapper.java [hunk 1]
```diff
@@ -102,8 +103,8 @@ private NetworkTypes getNetworkType() {
   }
 
   private void log(Events e, @NonNull BundleBuilder b) {
-    String deviceCountry = countryCode.getDeviceCountry().toUpperCase();
-    String networkCountry = countryCode.getNetworkCountry().toUpperCase();
+    String deviceCountry = countryCode.getDeviceCountry().toUpperCase(Locale.ENGLISH);
```
Comment (bemasc) [inline Android/app/src/main/java/app/intra/sys/firebase/AnalyticsWrapper.java]: Can you try "Locale.ROOT"?  That seems to be the one that provides standard string formatting behavior.

**Comment by bemasc** [comment]
> How do I test this?
> Is the CI test good enough?

The CI will build it but I doubt there's any test coverage of this method.  It seems like an extremely straightforward change, so I wouldn't worry about it.  If you want to test manually, we can try loading it in an emulator, changing to some strange locales, and adding some debug logging to check the results.

We could add automated tests for this function, but unless the tests run in a nonstandard locale they wouldn't catch the problem anyway, so I don't think it's worthwhile.

File: Android/app/src/main/java/app/intra/sys/firebase/AnalyticsWrapper.java (hunk 1 above)
Comment (fortuna) [reply Android/app/src/main/java/app/intra/sys/firebase/AnalyticsWrapper.java]: Done

**Pull request merged**